server/index.js

```diff
@@ -4,13 +4,46 @@ const bodyParser = require('body-parser')
 const dbFunctions = require('./db.js')
 const levelup = require('levelup')
 const db = levelup('./hint-hint')
+const moment = require('moment')
+const schedule = require('node-schedule')
+const twilio = require('twilio')
+const dotenv = require('dotenv').config()
+const client = new twilio(process.env.TWILIO_ACCOUNT_SID, process.env.TWILIO_AUTH_TOKEN)
 const app = express()
 
 app.use(bodyParser.json())
 
 const publicPath = path.join(__dirname, '/public')
 app.use(express.static(publicPath))
 
+schedule.scheduleJob('*/10 * * * *', () => {
+  const now = Date.now()
+  const promises = []
+  dbFunctions.getCollection(db, 'events')
+    .then((data) => {
+      for (let i = 0; i < data.length; i++) {
+        let event = data[i]
+        const time = moment(event.date).format('hh:mm a')
+        if (Math.abs(now - event.date) <= 3600000 && (now - event.date) < 0 && event.notified !== true) {
```

Comment:
ah, very nice. Let's extract that logic into another function, something like:

```js
function eventWithinHour(now, event) {
  return Math.abs(now - event.date) <= 3600000 && (now - event.date) < 0
}
```

This will make this check much easier to read:

```js
if (eventWithinHour(now, date) && !event.notified) {
...
}
```